app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class Header extends Component {
+  render() {
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route,
+    } = this.props;
+    return (
+      <>
+        <h1>Header</h1>
```

Comment:
This looks good, I think you are just missing passing props in your component call in App.js